buf.work.yaml

```diff
@@ -0,0 +1,8 @@
+# Generated by "buf config migrate-v1beta1". Edit as necessary, and
+# remove this comment when you're finished.
```

Comment:
comment makes it sound like the comment should already be gone, is that true,
or is the comment for later people?

I don't have the most context on this, so ignore if it's not a good question